desc/grid.py

```diff
@@ -662,23 +773,24 @@ def change_resolution(self, L, M, N, NFP=None):
             Number of field periods.
 
         """
-        self._NFP = NFP if NFP is not None else self.NFP
-        if L != self.L or M != self.M or N != self.N:
```

Comment:
Previously NFP could be changed without updating the Grid. But I don't think this makes sense because the node placement depends on NFP